.goreleaser.yml

```diff
@@ -12,6 +12,17 @@ builds:
       - linux
       - windows
       - darwin
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    goarm:
+      - 7
```

Comment:
I think it's better to drop this bit and use the default ARMv6, see also https://go.dev/doc/install/source#environment.

That will also work on ARMv7 and makes it work a bit more generally also for older ARM hardware. We're not trying to eek out the last bit of performance here then.

Reply:
```
GOARM=6: use VFPv1 only; default if cross compiling; usually ARM11 or better cores (VFPv2 or better is also supported)
```